index.html

```diff
@@ -240,7 +281,9 @@
                     <img id="closeGotoVoiceMessagePopupBTN" src="icons/voiceMessage/group.png"
                         onclick="closeGotoVoiceMessagePopup()" />
                 </div>
-            </div>
+            </div> -->
```

Comment:
delete - it's in the git history if you'll ever need it

Reply:
added back this code section 
this popup is needed